### ISSUE TITLE
[Bug]: Is release process broken?

### ISSUE THREAD

**Issue by nhoizey** [tokens-studio/sd-transforms#243]
### What happened?

It look like releases don't go to `npm` anymore, the latest available version is `0.12.2`.

### Expected output

For example here, a `0.13.0` should have been released: https://github.com/tokens-studio/sd-transforms/pull/242

### Version

0.12.2

**Comment by jorenbroekema** [comment]
Yes it's been broken for a while so I've been manually releasing. I just forgot about that detail after the xmas holidays, thanks for letting me know ;) it's released now!

**Comment by nhoizey** [comment]
@jorenbroekema awesome, thanks a lot! 🙏
